docs/source/pages/using.rst

```diff
@@ -168,3 +168,21 @@ Thanks to Datalog’s expressive language model, it’s easy to add exception ru
 requirement. For example, `the Mysql Connector/J <https://slsa.dev/spec/v0.1/requirements#build-service>`_ dependency in
 the Micronaut MuShop project does not pass the ``build_service`` check, but can be manually investigated and exempted if trusted. Overall, policies expressed in Datalog can be
 enforced by Macaron as part of your CI/CD pipeline to detect regressions or unexpected behavior.
+
+---------------------------
+Analyzing more dependencies
+---------------------------
```

Comment:
I think it would be better to move this section as a subsection of `Analyzing with an SBOM`. What do you think?